src/KubernetesClient/KubernetesClient.csproj

```diff
@@ -36,10 +36,13 @@
     <PackageReference Include="Portable.BouncyCastle" Version="1.8.1.3" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.10" />
     <PackageReference Include="prometheus-net" Version="4.1.1" />
+    <PackageReference Include="Serilog.Extensions.Logging" Version="2.0.2" />
```

Comment:
can you use default trace instead of serilog
want to keep less dependencies for end users

Reply:
it doesnt seem like `OidcClient` is compatible with `ServiceClientTracing` so i have removed the verbose logging. if you see a way to make this work please let me know and ill add it back
https://github.com/IdentityModel/IdentityModel.OidcClient/blob/main/src/OidcClient/OidcClient.cs